docs/visual-basic/misc/bc42207.md

```diff
@@ -1,20 +1,19 @@
 ---
 title: "Assembly reference '<reference>' is invalid and cannot be resolved"
 ms.date: 07/20/2015
-f1_keywords: 
+f1_keywords:
   - "vbc42207"
   - "bc42207"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "BC42207"
 ms.assetid: c6129559-7273-4abe-b2f1-deea05d242c9
 ---
 # Assembly reference '\<reference>' is invalid and cannot be resolved
-**Error ID:** BC42207  
-  
-## To correct this error  
-  
-1. 
-  
+
+**Error ID:** BC42207
+
+## To correct this error
```

Comment:
```suggestion
```

Reply:
I think we can keep this to try to either fill those later or remove these topics since they don't help with anything.